substrate/frame/membership/src/lib.rs

```diff
@@ -33,6 +33,15 @@ use sp_std::prelude::*;
 pub mod migrations;
 pub mod weights;
 
+#[cfg(any(feature = "mock", test))]
```

Comment:
Does this `mock` feature exist? I dont think its needed otherwise.

Reply:
Yes mock is the file where `mock` runtime is defined.